src/pages/MemberNoteAdminPage.tsx

```diff
@@ -67,22 +67,24 @@ const MemberProfileAdminPage: React.FC = () => {
           }
         />
         <AdminBlock className="mt-4">
-          {notes.length === 0 ? (
+          {notes.filter(note => !note.deletedAt).length === 0 ? (
```

Comment:
filter out deleted notes when fetching member_notes data.
deleted notes is shown at `NoteCollectionPage` now.

Reply:
should fix the select permission of the general member in hasura